provider/aws/formation/app.json.tmpl

```diff
@@ -240,6 +240,9 @@
             { "Key": "deregistration_delay.timeout_seconds", "Value": "30" },
             { "Key": "stickiness.enabled", "Value": "true" }
           ],
+          "Tags": [
+            { "Key": "ProcessName", "Value": "{{ upper .Name }}" }
```

Comment:
Let's call this `Service`. Might as well add `App` as well while you're in there.

Reply:
App is populated in tag Name from cloudformation tags, though I see no reason why not to add this to be more explicit

Reply:
Gotcha. If `App` is already in there don't worry about it.

Reply:
It is not called App, so for the clearness I have added the App as well. Not sure if used proper variables, though looking on other entries it should be ok.